tests/modules/ingestion-pipeline/propagation.test.ts

```diff
@@ -0,0 +1,55 @@
+import { beforeEachTruncateCollections } from "../../hooks/collections";
+import { beforeAllCreateEngines } from "../../hooks/engines";
+import { beforeEachLoadFixtures } from "../../hooks/fixtures";
+
+import { sendDummyTempPayloads, useSdk } from "../../helpers";
+import { MeasureContent } from "../../../lib/modules/measure";
+
+jest.setTimeout(10000);
+
+describe("Ingestion Pipeline: propagation", () => {
+  const sdk = useSdk();
+
+  beforeAll(async () => {
+    await sdk.connect();
+    await beforeAllCreateEngines(sdk);
+  });
+
+  beforeEach(async () => {
+    await beforeEachTruncateCollections(sdk);
+    await beforeEachLoadFixtures(sdk);
+  });
+
+  afterAll(async () => {
+    sdk.disconnect();
+  });
```

Comment:
You can encapsulate all that code above in a single function and call it each time your need:

```js
export function initSdk(): Kuzzle {
  const sdk = useSdk();

  beforeAll(async () => {
    await sdk.connect();
    await beforeAllCreateEngines(sdk);
  });

  beforeEach(async () => {
    await beforeEachTruncateCollections(sdk);
    await beforeEachLoadFixtures(sdk);
  });

  afterAll(async () => {
    sdk.disconnect();
  }); 

return sdk;
}
```

then use like:

`const sdk = initSdk();`